dist/js/L.DNC.js

```diff
@@ -480,6 +505,27 @@ L.DNC.Menu = L.Class.extend({
         }.bind(this));
     } ,
 
+    buildMenu : function() {
+        var domElement = L.DNC.Menu.prototype.buildMenu.call(this);
+
+        // Sort
+        var keys = [];
+        for (var key in this.ops.geom) {
+            keys.push(key);
+        }
+        keys = keys.sort();
+
+        // Build menu dropdown
+        html = '<div class="menu-dropdown menu-expand">';
+        for (var i=0; i < keys.length; i++) {
+            var key = keys[i];
+            html += '<button class="menu-button menu-button-action" id="' + key + '">' + key + '</button>';
+        }
+        html += '</div>';
+        domElement.appendChild(this._buildElement(html));
+        return domElement;
```

Comment:
As all menu classes will do something similar to this, this should probably be generalized and placed in `L.DNC.Menu`